web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -790,6 +780,9 @@ else if (IMPLEMENTATION_ID_SETUP.equals(page)) {
 							//If we have a runtime properties file, get the database setup details from it
 							if (skipDatabaseSetupPage()) {
 								Properties props = OpenmrsUtil.getRuntimeProperties(WebConstants.WEBAPP_NAME);
+								if (props == null) {
```

Comment:
Was there a need to mix this change with the demo data removal?

Reply:
My IDE warned and i said why not! Let me remove it now for a later fix i guess